tests/unit/components/materialize-collection-links-test.js

```diff
@@ -0,0 +1,32 @@
+import {
+  moduleForComponent,
+  test
+} from 'ember-qunit';
+
+moduleForComponent('materialize-collection-links', {
+  // Specify the other units that are required for this test
+  // needs: ['component:foo', 'helper:bar']
+});
+
+test('it renders', function(assert) {
+  assert.expect(2);
+
+  // Creates the component instance
+  var component = this.subject();
+  assert.equal(component._state, 'preRender');
+
+  // Renders the component to the page
+  this.render();
+  assert.equal(component._state, 'inDOM');
+});
+
+test('it is added to the page', function(assert) {
+  this.subject();
+  this.render();
+  assert.ok($('div').length);
```

Comment:
This seems to be a nonsensical test. You are checking for the presence of a div anywhere on the page.